src/main/java/hometask3/shape/Rectangular.java

```diff
@@ -0,0 +1,17 @@
+package hometask3.shape;
+
+public abstract class Rectangular implements Shape{
+    int a;
+    int b;
+
+    public Rectangular(int a) {
+        this.a = a;
+    }
+
+    public Rectangular(int a, int b) {
+        this.a = a;
+        this.b = b;
+    }
+
+    abstract int getSquare();
```

Comment:
initial calculation of the square should be done in abstract classes (and can be reused by other child-classes), if it is needed (e.g. for Circle and Square) - it should be overridden in the child class itself